### PR TITLE
Analyse and touchscreen

### LINKED ISSUE #242: touchevents on the table rows (homepage) don't allow any other touch events (scrolling, other button clicks) etc
Solution - When the user's device is a touch screen, show the analyse button next to play/edit/duplicate button 
If it's not a touch screen, the existing behaviour persists

### PR DESCRIPTION
**Pull request by deepansh96**
Fixes #242

### CODE REVIEW

File: src/components/Collections/Table/Table.vue [hunk 1]
```diff
@@ -238,7 +241,13 @@ export default {
     },
     tableRowClass() {
       // class for each row of the table
-      return "hover:bg-gray-100 hover:cursor-pointer active:bg-gray-100";
+      return [
+        {
+          "hover:bg-white": this.isTouchDevice,
```
Comment (dalmia) [inline src/components/Collections/Table/Table.vue:246]: is this needed? isn't this the default?

File: src/components/Collections/Table/Table.vue [hunk 2]
```diff
@@ -93,7 +93,6 @@
                   :class="tableRowClass"
```
Comment (dalmia) [inline src/components/Collections/Table/Table.vue:93]: Added one more issue to the list of bugs:
- All plios shows number of plios shown in table not total number of plios

Comment (dalmia) [reply src/components/Collections/Table/Table.vue:93]: Also, the other issue mentioned in the bug:
- Reduce size of URL in all plios home page to fit screen

will this be a separate PR?

File: src/components/Collections/Table/Table.vue [hunk 3]
```diff
@@ -93,7 +93,6 @@
...
                   @mouseover="tableRowHoverOn(rowIndex)"
                   @mouseout="tableRowHoverOff"
-                  @touchstart="tableRowTouchOn(rowIndex)"
```
Comment (dalmia) [inline src/components/Collections/Table/Table.vue:96]: Screenshot 2021-05-14 at 10 40 04 AM
breaks in 320 px screen.

**Review by dalmia — COMMENTED** [review]
Everything looks okay apart from the questions asked above - have tested locally.

File: src/components/Collections/Table/Table.vue (hunk 3 above)
Comment (deepansh96) [reply src/components/Collections/Table/Table.vue:96]: Looks fine on my side??

Comment (dalmia) [reply src/components/Collections/Table/Table.vue:96]: Screenshot 2021-05-14 at 12 29 56 PM
staging pe bhi fatt raha

File: src/components/Collections/Table/Table.vue (hunk 2 above)
Comment (deepansh96) [reply src/components/Collections/Table/Table.vue:93]: Add both of them in this PR

File: src/components/Collections/Table/Table.vue (hunk 3 above)
Comment (deepansh96) [reply src/components/Collections/Table/Table.vue:96]: Screenshot 2021-05-14 at 12 45 32 PM
Not that bad in my case
URL is long so thoda problem hai which will be fixed in the change I've made to reduce the size

File: src/components/Collections/ListItems/PlioListItem.vue [hunk 4]
```diff
@@ -235,6 +239,8 @@ export default {
         return "";
       }
       var baseURL = process.env.VUE_APP_FRONTEND + "/#";
+      baseURL = baseURL.replace("http://", "");
+      baseURL = baseURL.replace("https://", "");
```
Comment (dalmia) [inline src/components/Collections/ListItems/PlioListItem.vue:243]: function for this.

**Pull request merged**